doc/nrf/device_guides/wifi_coex.rst

```diff
@@ -86,9 +86,6 @@ Supported implementations
 
 The following are the implementations supported by the MPSL-provided Bluetooth-only coexistence.
 
-.. note::
```

Comment:
@wiba-nordic Should this change be backported to older NCS releases?

Reply:
On a general level, we don't make changes to old releases. If we make patch releases based on them, this could be something to add those.